webapp/src/views/projects/translations/cell/StateTransitionButtons.tsx

```diff
@@ -30,9 +28,12 @@ export const StateTransitionButtons: React.FC<Props> = ({
           data-cy="translation-state-button"
           onClick={() => onStateChange?.(nextState)}
           className={className}
-          tooltip={t('translation_state_change', {
-            newState: t(translationStates[nextState]?.translationKey || ''),
-          })}
+          tooltip={
+            <>
+              <T keyName="translation_state_mark_as" />{' '}
+              {translationStates[nextState]?.translation}
```

Comment:
Wouldn't this be problematic for languages where these would have to be in a different order?

Reply:
Ah, yeah, guess you are right.